### DISCUSSION TITLE
Link to ScriptLogs for email alert

### DISCUSSION THREAD

**Discussion by trieppo** [ptarmiganlabs/butler#1214] [Ideas]
When a task fails - it would be awesome to construct a link to the actual physical log file in Butler using JavaScript etc. in failed-reload.handlebars.

We have some cases (SAP Connector) where the error is not in the last 15 lines. 

If a user had a link pointing to the file system - they could download and read the entire script log.

**Comment by mountaindude** [comment]
Good idea. 
You can choose to include more than 15 lines by editing the config file, but even so your suggestion is good.
Without looking into it I can say how easy/difficult it would be to implement this, but as always - anything is possible until proven otherwise...

**Comment by mountaindude** [comment]
@trieppo I am thinking about this ticket, but not sure what the link should look like. 
It has to be something that works from an email, Teams or Slack message.

Something like \\host.domain.net\path\to\logfiles\file.txt ?
Not sure how that works from an email/Teams/Slack though.. Would have to test. 

Any ideas/suggestions?

**Comment by trieppo** [comment]
Hey!

Archived Log files are on a Windows file share.

So \\<share>\Logs\script\

Don't remember exactly now. I can check later.
